slang-gfx.h

```diff
@@ -41,6 +46,10 @@ typedef SlangResult Result;
 typedef SlangInt Int;
```

Comment:
Let's define this to be `int32_t` and `uint32_t` for now.

Reply:
I tried fiddling around with this change and it causes some major issues because there's also a `typedef SlangInt Int;`, which results in a large number of ambiguous symbol errors in some of my test files. I think this would be better left as a separate PR, especially since many of these could potentially be replaced with `Index` or `Count`.

Reply:
I see. Can we at least make Index and Count 32bit then? I know Tess likes 64bit ints, but I am worried that using 64bit across the board could make things slower due to their larger memory footprint.

Reply:
Also causes similar issues at the moment. There's a typedef in `slang-common.h` for both `Index` and `Count`. I think it'd still be better to leave this for a separate PR and properly hash out how to handle all these integer types and what to typedef them as first.

Reply:
Ahh never mind, I'll just make the change here since nothing will actually be affected by this typedef rename. I'll leave the actual renaming changes throughout the rest of the header for later.